lib/noid/minter.rb

```diff
@@ -93,7 +95,16 @@ def counters
     end
 
     def dump
-      { state: true, seq: @seq, seed: @seed, template: template.template, counters: Marshal.load(Marshal.dump(counters)) }
+      {
+        template: template.template,
+        counters: Marshal.load(Marshal.dump(counters)),
```

Comment:
Ah, the line scrolled off the screen and I missed that this is preserving old behavior. Moving right along.